### PR TITLE
#81 - Fix ITCase

### LINKED ISSUE #81: Helm integration tests failing
`HelmCompatibilityITCase#helmRepoAddAndUpdateWorks` test started to fail because of the following error:
```
STDERR:
Error: Looks like "https://kubernetes-charts.storage.googleapis.com" is not a valid chart repository or cannot be reached: Failed to fetch https://kubernetes-charts.storage.googleapis.com/index.yaml : 403 Forbidden
```
The problem is related to the fact that https://kubernetes-charts.storage.googleapis.com/index.yaml as default stable repository is no longer allowed (https://stackoverflow.com/a/65404574/14676855). It is necessary to fix this problem.

### PR DESCRIPTION
**Pull request by genryxy**
Closes #81

### CODE REVIEW

File: src/test/java/com/artipie/helm/HelmCompatibilityITCase.java [hunk 1]
```diff
@@ -93,7 +93,7 @@
     /**
      * The turl.
      */
-    private String turl;
+    private String url;
```
Comment (olegmoz) [inline src/test/java/com/artipie/helm/HelmCompatibilityITCase.java:96]: @genryxy please provide better description for this field

**Review by olegmoz — CHANGES_REQUESTED** [review]
@genryxy thanks, please see one comment

**Comment by genryxy** [comment]
@olegmoz thanks, corrected

**Review by olegmoz — APPROVED** [review]
@genryxy thanks, merging

**Comment by 0crat** [comment]
Job `gh:artipie/helm-adapter#82` is not assigned, can't get performer

**Pull request merged**